crates/net/network/src/manager.rs

```diff
@@ -182,7 +182,8 @@ where
             dns_discovery_config,
             #[cfg(feature = "optimism")]
             sequencer_endpoint,
-            ..
+            #[cfg(feature = "optimism")]
+            disable_tx_gossip,
```

Comment:
I think we may have to keep the spread operator here, but haven't verified with linting.
```suggestion
            #[cfg(feature = "optimism")]
            disable_tx_gossip,
            ..
```

Reply:
I didn't see any failures but but not sure I ran all the right checks.  Anything I can run to confirm or should I just revert this?

Reply:
This looks fine - passed CI's lint jobs.